src/components/Person/Person/Person.jsx

```diff
@@ -0,0 +1,28 @@
+export const Person = ({
+  person: {
+    name,
+    age,
+    isMarried,
+    partnerName,
+    sex,
+  }
+}) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
```

Comment:
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
      
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
      
      <p className="Person__partner">
```
Don't forget to add newlines between blocks of code to improve readability